pom.xml

```diff
@@ -153,6 +153,12 @@
 
         <groovy.version>2.4.8</groovy.version>
 
+        <asciidoctor-maven-plugin.version>1.5.3</asciidoctor-maven-plugin.version>
+        <jruby.version>1.7.26</jruby.version>
+        <asciidoctorj.version>1.5.4.1</asciidoctorj.version>
+        <asciidoctorj-pdf.version>1.5.0-alpha.11</asciidoctorj-pdf.version>
+        <asciidoctorj-epub3.version>1.5.0-alpha.6</asciidoctorj-epub3.version>
```

Comment:
Yeah, it does not do any harm so I let it there as this way, we can enable it quickly to do more tests if we want to.